### DISCUSSION TITLE
class in demo

### DISCUSSION THREAD

**Discussion by yairifrachi** [clientIO/joint#2664] [Q&A]
### Introduction

i I use your demo service for learning purposes.
But I see that the examples are only in react class
Are the examples also in react functions?
I mean to KitchenSink in react TS

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
The `KitchenSink` app still needs to migrated from classes to functions.

An example of React functions is shown in our tutorial: https://resources.jointjs.com/tutorial/react-ts
